openpifpaf/network/losses/composite.py

```diff
@@ -82,6 +87,10 @@ def _confidence_loss(self, x_confidence, t_confidence):
         ce_loss = self.confidence_loss(x_confidence, bce_target)
         if self.prescale != 1.0:
             ce_loss = ce_loss * self.prescale
+        weight = torch.ones_like(t_confidence, dtype=torch.float, requires_grad=False)
```

Comment:
Here you don't need to specify dtype. It will obtain it from t_confidence. Remove it and then it will do the right thing no matter whether you are working with float32 or float64.